docs/reference/inference/elastic-infer-service.asciidoc

```diff
@@ -117,7 +104,7 @@ PUT /_inference/chat_completion/chat-completion-endpoint
 {
     "service": "elastic",
     "service_settings": {
-        "model_id": "model-1"
+        "model_id": ".rainbow-sprinkles-elastic"
```

Comment:
I think this should be `.rainbow-sprinkles`

The default inference endpoint would be `.rainbow-sprinkles-elastic` but we don't need to include that here.

We could have another example that leverages the default inference endpoint if we want though:

```
POST /_inference/chat_completion/.rainbow-sprinkles-elastic
{
  "messages: [...]
...
}
```